packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -73,7 +67,14 @@ export const ContextMenu: React.FC<ContextMenuProps> = React.memo(({ x, y, onClo
                 target={item.target}
                 icon={item.icon}
                 active={item.active}
-                onClick={onClick}
+                onClick={() => {
+                  if (item.onClick) {
+                    item.onClick();
+                  }
+                  if (onClose) {
+                    onClose();
+                  }
+                }}
```

Comment:
Could this work in this scenario?
```
onClick={item.onClick || onClose}
```

Reply:
@thisisobate  No, that is not the same as the above code. Read the above. What happens to your suggestion when onClick is set? Will onClose be called?

Reply:
oh yeah true 😅👍
You're right Torkel